sql/sql_class.h

```diff
@@ -2431,6 +2431,7 @@ class THD :public MDL_context_owner,
     default values only if (innodb_was_used==TRUE)
   */
   ulonglong  innodb_trx_id;
+  ulonglong  saved_innodb_trx_id;
```

Comment:
Assuming it is needed, then it probably needs some sort of initialisation/reset in THD or it looks like some of the reads will be uninitialized